src/torchmetrics/image/fid.py

```diff
@@ -322,7 +323,11 @@ def __init__(
                     f"Integer input to argument `feature` must be one of {valid_int_input}, but got {feature}."
                 )
 
-            self.inception = NoTrainInceptionV3(name="inception-v3-compat", features_list=[str(feature)])
+            self.inception = NoTrainInceptionV3(
```

Comment:
you need to provide a default to preserve actual behavior...

Reply:
The default is None and it preserves the actual behavior:
See https://github.com/toshas/torch-fidelity/blob/master/torch_fidelity/feature_extractor_inceptionv3.py#L31